### ISSUE TITLE
hexo init blog1 failed

### ISSUE THREAD

**Issue by shangsh** [hexojs/hexo#2769]
## Environment Info
Ubuntu 16.04
node 8.5.0
npm 5.3.0
I use http proxy 

## For BUG
shang@Z580:~/work/blog/hexo$ hexo init blog1
INFO  Cloning hexo-starter to ~/work/blog/hexo/blog1
Cloning into '/home/shang/work/blog/hexo/blog1'...
remote: Counting objects: 59, done.
remote: Total 59 (delta 0), reused 0 (delta 0), pack-reused 59
Unpacking objects: 100% (59/59), done.
Checking connectivity... done.
Submodule 'themes/landscape' (https://github.com/hexojs/hexo-theme-landscape.git) registered for path 'themes/landscape'
Cloning into 'themes/landscape'...
remote: Counting objects: 785, done.
remote: Total 785 (delta 0), reused 0 (delta 0), pack-reused 784
Receiving objects: 100% (785/785), 2.54 MiB | 953.00 KiB/s, done.
Resolving deltas: 100% (403/403), done.
Checking connectivity... done.
Submodule path 'themes/landscape': checked out 'decdc2d9956776cbe95420ae94bac87e22468d38'
INFO  Install dependencies
npm ERR! code EPROTO
npm ERR! errno EPROTO
npm ERR! request to https://registry.npmjs.org/hexo failed, reason: write EPROTO 140218982328128:error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol:../deps/openssl/openssl/ssl/s23_clnt.c:797:
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/shang/.npm/_logs/2017-09-24T23_18_22_208Z-debug.log
WARN  Failed to install dependencies. Please run 'npm install' manually!

**Comment by NoahDragon** [comment]
Could you please inside the `blog1` folder then run `npm install`?

**Comment by rafr3** [comment]
I have the same problem. 
First, tried to run `hexo init`, when it failed I tried to run `npm install` inside directory manually.
I'm not using proxy, I have access to registry url using any browser.
When I changed the registry from *https* to *http* then `hexo init .` works but that's not solution.

![obraz](https://user-images.githubusercontent.com/8841983/35770732-06897c44-0921-11e8-9f9d-6f39395250f1.png)
